flink-autoscaler-standalone/src/main/java/org/apache/flink/autoscaler/standalone/config/AutoscalerStandaloneOptions.java

```diff
@@ -121,4 +121,11 @@ private static ConfigOptions.OptionBuilder autoscalerStandaloneConfig(String key
                                             code(EVENT_HANDLER_TYPE.key()),
                                             code("JDBC"))
                                     .build());
+
+    public static final ConfigOption<Duration> JDBC_EVENT_HANDLER_TTL =
+            autoscalerStandaloneConfig("jdbc-event-handler.ttl")
```

Comment:
how about `jdbc.event-handler.ttl`?

Other options are `jdbc.xxx`.

Reply:
thanks for the comments.
> jdbc.event-handler.ttl
Let's do it this way.
Consistent naming conventions are better!